### PR TITLE
Remove relative references

### PR DESCRIPTION
**Pull request by IreneMLC**
### Relevant information

- Remove relative references from the definition.yml files
- Add schema validation for the name of the dashboard and composite metric files.

### CODE REVIEW

File: validator/schemas/entity-schema-v1.json [hunk 1]
```diff
@@ -211,6 +211,10 @@
     "dashboardTemplates": {
       "$id": "#/properties/dashboardTemplates",
       "type": "array",
+      "items": {
+        "type": "string",
+        "pattern": "^([aA-zZ_-])*(dashboard.json)$"
```
Comment (naxhh) [inline validator/schemas/entity-schema-v1.json:216]:
```suggestion
        "pattern": "^([a-zA-Z_-])*(dashboard.json)$"
```

Is this what you where looking for? 

The current regex will match [, \, ], ^, _, and `

Comment (naxhh) [reply validator/schemas/entity-schema-v1.json:216]: Also supper nit but the regex doesn't need the capture groups `^[aA-zZ_-]*dashboard.json$`

**Review by naxhh — APPROVED** [review]
LGTM aside of the regex comment.

Nice!

File: validator/schemas/entity-schema-v1.json (hunk 1 above)
Comment (IreneMLC) [reply validator/schemas/entity-schema-v1.json:216]: Thanks! you're right!

**Comment by IreneMLC** [comment]
Already discussed this with @srvaroa offline, so I'll go ahead and merge to unblock ongoing work.

**Pull request merged**